burnman/anisotropy.py

```diff
@@ -0,0 +1,455 @@
+# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
+# Copyright (C) 2012 - 2017 by the BurnMan team, released under the GNU
+# GPL v2 or later.
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from .tools import unit_normalize
+from .material import Material, material_property
+
+class AnisotropicMaterial(Material):
```

Comment:
At the moment, we get very little benefit, but we will eventually get a lot out of this, when we have an EoS to describe how Cijs change with pressure and temperature. I think it's valuable to prepare for this now.